tools/qontract_cli.py

```diff
@@ -3680,14 +3680,18 @@ def get_promotion_state(channel: str, sha: str):
         SaasPromotionState,
     )
 
+    bucket = os.environ.get("APP_INTERFACE_STATE_BUCKET")
     promotion_state = SaasPromotionState.create(promotion_state=None, saas_files=None)
     for publisher_id, state in promotion_state.get(channel=channel, sha=sha).items():
         print()
         if not state:
             print(f"No state found for {publisher_id=}")
         else:
-            print(f"State for {publisher_id=}:")
-            print(state)
+            print(f"{publisher_id=}")
+            print(
+                f"State link: https://us-east-1.console.aws.amazon.com/s3/object/{bucket}?region=us-east-1&bucketType=general&prefix=state/openshift-saas-deploy/promotions_v2/{channel}/{publisher_id}/{sha}"
```

Comment:
Can we get region from env var? `us-east-1` can be default if not provided

Reply:
Good idea. Added from env var

Reply:
@fishi0x01 missed one replace in string `?region=us-east-1`

Reply:
Good catch - adding here https://github.com/app-sre/qontract-reconcile/pull/4455